src/amo/components/CollectionManager/index.js

```diff
@@ -37,7 +38,18 @@ import type { ReactRouterType } from 'core/types/router';
 
 import './styles.scss';
 
+export const ADDON_ADDED_STATE_PENDING: 'ADDON_ADDED_STATE_PENDING'
+  = 'ADDON_ADDED_STATE_PENDING';
+export const ADDON_ADDED_STATE_SUCCESS: 'ADDON_ADDED_STATE_SUCCESS'
+  = 'ADDON_ADDED_STATE_SUCCESS';
+
+export type AddonAddedStatusType = |
+  typeof ADDON_ADDED_STATE_PENDING |
+  typeof ADDON_ADDED_STATE_SUCCESS |
+  null;
```

Comment:
We put the pipe first usually :)

```js
export type AdonAddedStatusType =
  | ...
  | ...;
```

Reply:
I would not put `null` here by the way because the state (or status) should really be either "pending" or "success". The state attribute could be of that type *or* `null`, and we could reuse this type somewhere else, maybe.